cookbook/deployment/customizing_resources.py

```diff
@@ -38,7 +38,7 @@
 
 
 # %%
-# Define a task and configure the resources to be allocated to it.
+# Define a task and configure the resources to be allocated to it. Currently, you can specify the maximum memory to be ``1Gi`` in the demo cluster.
```

Comment:
I think we also need to talk about how to change that limit. Do we have that document?

Reply:
I think we can document it or make it as a note in a different place. I can look at it.. @samhita-alla